packages/sync/src/class-defaults.php

```diff
@@ -1160,7 +1160,7 @@ public static function is_multi_network() {
 	 *
 	 * @var int Number of seconds.
 	 */
-	public static $default_cron_sync_time_limit = 30; // 30 seconds.
+	public static $default_cron_sync_time_limit = 4 * MINUTE_IN_SECONDS; // 30 seconds.
```

Comment:
You need to fix or remove this comment :)

```php
// 30 seconds.
```

Reply:
```suggestion
	public static $default_cron_sync_time_limit = 4 * MINUTE_IN_SECONDS;
```